web-ui/src/components/feedback_recipient_selector/FeedbackRecipientSelector.jsx

```diff
@@ -1,20 +1,21 @@
 import React, { useContext, useEffect, useState, useRef } from 'react';
 import { styled } from '@mui/material/styles';
-import './FeedbackRecipientSelector.css';
```

Comment:
I don't think Sonar can really detect whether this import is used. We probably need to restore all CSS imports.

Reply:
Yeah, I tried to arrange them all to kind of match each other... In the main the css import comes just prior to the const values

Reply:
That matches my preference.